doc/guides/dts/index.rst

```diff
@@ -341,10 +347,10 @@ hierarchy looks like the following::
   dts/bindings/
   boards/<ARCH>/<BOARD>/
 
-The common directories contain a skeleton.dtsi include file that defines the
-address and size cells.  The yaml subdirectory contains common yaml files for
-Zephyr-specific nodes/properties and generic device properties common across
-architectures.
+The common directory contains a skeleton.dtsi which provides device tree root
+node definition.  The bindings subdirectory contains YAML binding files used
+to instruction how python DTS parsing script should extract nodes information
```

Comment:
Something is wrong with the english:

"used to instruction how"

Reply:
Was supposed to be ""used to instruct how"". Txs